README.md

```diff
@@ -4,6 +4,8 @@ Small server wrapper around Z-Wave JS to access it via a WebSocket.
 
 ## Trying it out
 
+Keep in mind that on production you want to execute `npm run build` and use `node` instead of `ts-node` for all bellow commands.
```

Comment:
```suggestion
Keep in mind that on production you want to execute `npm run build` and use `node` instead of `ts-node` for all below commands.
```

Reply:
I don't htink that this sentence makes sense.

In production, when installed from `npm`, it is already JS and they should use the CLIs that we install: 

https://github.com/zwave-js/zwave-js-server/blob/master/package.json#L6-L9

Reply:
That's true if installed via `npm`, but it might be worth to mention that `ts-node` is not preferred for running it in long term if someone decide to clone.